scripts/portal.js

```diff
@@ -135,7 +135,17 @@ function Portal(url)
     html += "<br />"
     // Version
     if(this.json.client_version){
-      html += "<span class='version "+(this.json.client_version == r.home.portal.json.client_version ? 'same' : '')+"'>"+this.json.client_version+"</span>"
+      // Used to check if the rotonde version matches when multiple version lines are present.
+      var client_version_main = this.json.client_version.split(/\r\n|\n/)[0];
+      // Based off of entry.formatter
```

Comment:
I wonder if the formatting functionality in `entry.js` couldn't be reused for version formatting

Reply:
I thought about it as well, and as more content needs to be escaped / formatted, is moving `escape_html` to Rotonde (accessible via `r.escape_html`) an okay solution? I don't know where else it'd fit.

Reply:
Actually, I'm currently planning on "exposing" `r.escape_html` and `r.escape_attr`  to be used for "inner HTML" and attributes, respectively.